domain-server/src/NodeConnectionData.cpp

```diff
@@ -51,12 +51,12 @@ NodeConnectionData NodeConnectionData::fromDataStream(QDataStream& dataStream, c
 
     dataStream >> newHeader.lastPingTimestamp;
     
-    SocketType publicSocketType, localSocketType;
+    quint8 publicSocketType, localSocketType;
```

Comment:
Why this change? SocketType isn't a Qt concept.

Reply:
This and related changes in commit `058b9032` fixes Qt 5.12 not serializing/deserializing to these variables using the `>>` and `<<` operators.